Bottom: d73439e23062057d25cda6528eef924176daed66
Top:    cd1bdd70f84a7fb7711241c5664f362a049d7391
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 04:01:23 +0000

TEMPORARY: Comment out some DrawMark calls which might be out of context

Should fix this properly with notificatons to the GUI


---

diff --git a/src/action.c b/src/action.c
index 88907af..154d170 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4020,7 +4020,7 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
     {
       if (Marked.status)
 	{
-	  DrawMark (true);
+//	  DrawMark (true);
 	  Marked.status = false;
 	}
       else
@@ -4028,16 +4028,16 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
 	  Marked.status = true;
 	  Marked.X = Crosshair.X;
 	  Marked.Y = Crosshair.Y;
-	  DrawMark (false);
+//	  DrawMark (false);
 	}
     }
   else if (GetFunctionID (function) == F_Center)
     {
-      DrawMark (true);
+//      DrawMark (true);
       Marked.status = true;
       Marked.X = Crosshair.X;
       Marked.Y = Crosshair.Y;
-      DrawMark (false);
+//      DrawMark (false);
     }
   return 0;
 }
